spec/features/merchants/dashboard_spec.rb

```diff
@@ -211,6 +211,7 @@
       invoice_item_6 = item.invoice_items.create(invoice_id:invoice_6.id, quantity:3, unit_price: 3000)
 
       visit "/merchants/#{merchant.id}/dashboard"
+      save_and_open_page
```

Comment:
keep it dry. no save and open pages left in for the main.

Reply:
@DrewProebstel  hey just testing if you get a notification for this message